wordpress-mobile/WordPress-iOS
# Prompt user to logout before connecting to WPCom site

Fixes #5166 

When a user chooses to add a self hosted site but inputs a WPCom site address, we check to see is a default WPCom account already exists in the app
and if it does we prompt the user to logout before connecting to the other site.

![selfhostedprompt](https://user-images.githubusercontent.com/1335657/51447366-4a7e1e80-1cd2-11e9-95e0-9cd6722d63a2.gif)


To test:
1. Start by being signed in to a WordPress.com account.
2. Go to the My Sites tab and view the list of sites.
3. Tap the + button in the nav bar and choose "Add self-hosted site"
4. On the site address screen, enter the URL for a WordPress.com site that does not belong to the WordPress.com account that's currently logged into the app.
5. On the next screen, enter the WordPress.com credentials for the account that owns the site at the URL entered on the previous screen.
6. Repeat steps 1-5 with a site that is not WPCom.
7. See that the alert does not show and you are pushed to the credentials screen.

Update release notes:

* Added alert to prompt the user to logout before connecting to a wordpress.com site, if they are already logged in with a wordpress.com account
